### ISSUE TITLE
Please add -w wait until all pages are locked to manual

### ISSUE THREAD

**Issue by tycho-kirchner** [hoytech/vmtouch#90]
Hi,
and thanks for this tool (:
The important option -w misses in the [manual](https://github.com/hoytech/vmtouch/blob/master/vmtouch.pod) and is only displayed when issuing --help. This is confusing. After all, in daemon mode intuitively I expected a non-zero exit code on error, which is only the case, if -w is passed. I think this should be changed, if a user is not interested in the return value, he could simply launch vmtouch async using the ampersand &.

Thanks
Tycho

**Comment by hoytech** [comment]
I agree on all counts. In fact, I would like to make -d imply -w if possible. Are there any backwards-compatibility problems if we do that?

About adding -w to the manual, can you please send in a pull request? Thank you!

**Comment by cy7yz2rj** [comment]
Being left out of the manpage means I've only now discovered the -w option.
I had a script for launching discrete vmtouch -dL instances that relied on -w being absent; each vmtouch process would read from a different physical HDD so they could all run concurrently.

If I understand correctly, making -d imply -w would make my vmtouch invocations run sequentially instead of simultaneously. I'm not against the change, in fact I've already updated my script to use -dwL, just pointing out a possible scenario where it matters.
